joplin/api/schema.py

```diff
@@ -359,6 +359,49 @@ class Meta:
         interfaces = [graphene.Node]
 
 
+class DepartmentPageTopServiceNode(DjangoObjectType):
+    service_page = graphene.Field(ServicePageNode)
+    information_page = graphene.Field(InformationPageNode)
+    guide_page = graphene.Field(GuidePageNode)
+
+    def resolve_service_page(self, info):
```

Comment:
This is a graphene thing, since we have service_page defined as a graphene Field with the ServicePageNode type, graphene looks for a resolver (https://docs.graphene-python.org/en/latest/types/objecttypes/) and uses that to return the correct data when the API is called.

Reply:
cool, I'd love to find a good place to leave it as a note in the code